src/components/time-picker/_time-picker.scss

```diff
@@ -0,0 +1,43 @@
+//-----------------------------
+// Time Picker
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/import-once';
+
+@include exports('time-picker') {
+  .bx--time-picker {
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .bx--time-picker .bx--select-input {
+    padding-right: rem(32px);
+  }
+
+  .bx--time-picker .bx--select__arrow {
+    right: .875rem;
+  }
+
+  .bx--time-picker__input-field {
+    @include reset;
+    @include helvetica;
+    @include font-size('14');
+    display: flex;
+    align-items: center;
+    height: rem(40px);
+    color: $text-01;
+    background-color: $field-01;
+    border: 1px solid transparent;
+    width: 4.8rem;
+    color: $text-01;
+    height: rem(40px);
```

Comment:
You have `height` declared twice.
Also, a bit nitpicky but the `input` is *slightly* shorter than the two select components. If you change this to `height: 2.6rem` then it's 👌